javascript/javascript3/week1/gify.html

```diff
@@ -0,0 +1,30 @@
+<html>
```

Comment:
week1 stuff in the wrong folder. I'm only checking week2. If you want to have your week1 stuff graded, please make a separate PR with only that.